cfme/configure/tasks.py

```diff
@@ -160,3 +160,7 @@ def all_other_tasks(**filter_kwargs):
     Returns: List of dicts.
     """
     return _get_tasks("tasks_all_other", **filter_kwargs)
+
+
+def sort_by_header(header, order):
```

Comment:
@psav , @mfalesni, In cfme/configure/tasks.py, tasks_table is an instance of CheckboxTable.
The CheckboxTable class has no sorting methods.The cfme/web_ui/__init__.py file
also has this: class SortTable(Table). I'd like to create a new class, SortCheckboxTable and then use
tasks_table as an instance of SortCheckboxTable.

I'd appreciate your thoughts on this.